docs/reference/operators/index.mdx

```diff
@@ -6,7 +6,32 @@ description: Documentation for Operators
 
 import DocCardList from '@theme/DocCardList';
 
-TODO: document different operators:
-- description
-- what they provide? (kind of enrichment, etc)
-- parameters they take
+Operators are a very important part of the Inspetkor Gadget architecture. An
+operator is a component that performs a specific action, for example, loading
+eBPF programs, sorting the output, etc. Operators are modular and they all share
+the same interface. Currently all operators are shipped within Inspektor Gadget,
+but we plan to support user-provided operators in the future.
+
+Operators are configured in a pipeline, the data is generated on the layer
+operators and then passed through the differnt data operators until it's
+exported by one of them.
+
+### Layer Operators
```

Comment:
The code uses "image" and here we are talking about "layer" operators.

Reply:
No strong preference - I've used both over the course of the refactorings 😄 And both have pros and cons, IMHO.

Reply:
Let's use "layer" here then.